### ISSUE TITLE
GLFW.wheelPos is uninitialized/unused in 3.1.19

### ISSUE THREAD

**Issue by Deamon87** [emscripten-core/emscripten#17706]
**Version of emscripten/emsdk:**
3.1.19

While debugging, I noticed that `GLFW.wheelPos` in [library_glfw.js 3.1.19](https://github.com/emscripten-core/emscripten/tree/3.1.19/src/library_glfw.js) has `NaN` value in runtime. I believe it's because it's not initialized anywhere. 

(Probably it's not even used anymore)

**Comment by sbc100** [comment]
Indeed, looking at the history of that file it looks like this variable stopped being initialized back in bb637098a951fd0c4082e6720049e0de82aea15a .. which landed in 2014.   I guess nobody is depending on it?  Do you have code the is using this?

**Comment by Deamon87** [comment]
I dont personally. I noticed this only bcuz  there was a change in how ScrollCallback works between 1.x and 3.x (value_in_1_x = value_in_3_x * (-100)) and I had to debug to get what has changed. That's where I noticed `GLFW.wheelPos`. 

I see though it's used in that file on line 526 for GLFW2, but I guess noone uses GLFW2 anymore?
